Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -46,7 +46,14 @@ function InternalInit
     Assert ( $script:codeReviewClient -ne $null ) "Internal error: could not retrieve the CodeReviewHttpClient object"
                  
     Write-Verbose "Fetching data from build variables"
-    $repositoryId = GetTaskContextVariable "build.repository.id"    
+    $repositoryId = GetTaskContextVariable "build.repository.id"
+
+    # There is a bug in core agent 2.103.0 and lower   
+    if ($repositoryId -eq [Guid]::Empty)
```

Comment:
There's (![String]::IsNullOrWhiteSpace($repositoryId)) below, can you just use this here? You probably don't need the asserts since you are aware of case in which this is null....